src/libexpr/eval.cc

```diff
@@ -1094,13 +1095,32 @@ void EvalState::callPrimOp(Value & fun, Value & arg, Value & v, const Pos & pos)
     }
 }
 
+void fnStart(const Pos & pos) {
```

Comment:
Maybe mark these functions as `static inline` just to be sure.

Reply:
Done!